Fix compilation error, due to src/fr-enum-types.c which was bad generated using
glib-mkenums. "../../src/typedefs.h" didn't exist.

 src/fr-enum-types.c |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/fr-enum-types.c b/src/fr-enum-types.c
index df260be..8418183 100644
--- a/src/fr-enum-types.c
+++ b/src/fr-enum-types.c
@@ -1,10 +1,10 @@
 
 /* Generated data (by glib-mkenums) */
 
-#include <glib-object.h>
+/* enumerations from "typedefs.h" */
+#include "fr-enum-types.h"
+#include "typedefs.h"
 
-/* enumerations from "../../src/typedefs.h" */
-#include "../../src/typedefs.h"
 GType
 fr_compression_get_type (void)
 {
